app/containers/FeaturePage/index.js

```diff
@@ -17,17 +17,15 @@ import styles from './styles.css';
 class FeaturePage extends React.Component {
   constructor() {
     super();
-    this.onChangeRoute = this.onChangeRoute.bind(this);
-    this.changeRouteToHome = this.changeRouteToHome.bind(this);
   }
 
-  onChangeRoute(url) {
+  onChangeRoute = url => {
```

Comment:
I know, it's nice to have them to explain intent and maybe somebody wants to add multiple params later on.